lib/rig/event_stream/kafka_to_http.ex

```diff
@@ -2,6 +2,8 @@ defmodule Rig.EventStream.KafkaToHttp do
   @moduledoc """
   Forwards all consumed events to an HTTP endpoint.
 
+  TODO: do we want to keep this feature?
```

Comment:
good question... it was only needed this one time, and even then it wasn't a particularly beautiful solution. I guess we can remove it, but probably it makes sense to do this in a dedicated PR.